nexus_constructor/component_type.py

```diff
@@ -101,6 +101,8 @@
 }
 
 CHOPPER_CLASS_NAME = "NXdisk_chopper"
+SOURCE_CLASS_NAME = "NXsource"
+SAMPLE_CLASS_NAME = "NXsample"
```

Comment:
Could you check for occurrences of "NXsample" in the repo (except in component_type.py) and replace with SAMPLE_CLASS_NAME.